docs/commands/linux_libc_elf/got.md

```diff
@@ -29,5 +23,13 @@ Examples:
 |-a|--all|Process all libs/obfjiles including the target executable.|
 |-r|--show-readonly|Also display read-only entries (which are filtered out by default).|
 
+### Examples
+```text
+got
+got puts
+got -p libc
+got -a
```

Comment:
Is this useful at all not showing output? Also not sure what -p -a means, i mean, i know its described above, heh...